path_role.go

```diff
@@ -440,6 +447,10 @@ func (b *jwtAuthBackend) pathRoleCreateUpdate(ctx context.Context, req *logical.
 		role.VerboseOIDCLogging = verboseOIDCLoggingRaw.(bool)
 	}
 
+	if maxAgeRaw, ok := data.GetOk("max_age"); ok {
+		role.MaxAge = time.Duration(maxAgeRaw.(int)) * time.Second
```

Comment:
What about allowing for formatted strings such as `"1h"`? You could use [`parseutil.ParseDurationSecond`](https://github.com/hashicorp/vault/blob/master/sdk/helper/parseutil/parseutil.go#L17) for this.

Reply:
Using duration strings is already supported by way of the field being a `TypeDurationSecond`. You can see that the tests in this PR provide some duration strings like `2m`, `1hr`. At this line, `maxAgeRaw` is the integer number of seconds that's already parsed from a given duration string (see [field_data.go#L213](https://github.com/hashicorp/vault/blob/master/sdk/framework/field_data.go#L213)).

Reply:
Oh interesting, I didn't realize the router did that conversion for us.

Reply:
Yep, it's pretty convenient!